examples/common/pigweed/device_common_service/device_common_service.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
```

Comment:
nitpicky similar comment about stuttering/common. How about `device/service.proto`.

Reply:
Removed the common.

Reply:
Removing common seems better, but still doesn't have quite the brevity that was suggested.  Could this work?

`examples/common/pigweed/service/button.proto`
`examples/common/pigweed/service/device.proto`

Reply:
How about: 
common/pigweed/protos/empty.proto
common/pigweed/protos/button_service.proto
common/pigweed/protos/device_service.proto

Reply:
Done